pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -143,17 +199,26 @@ export default (
     },
 
     /**
-     * Emit on input with delay
-     * Note: Arrow function is avoided due context binding
+     * Emit on input with delay. Note: Arrow function is avoided due context 
+     * binding.
      */
     delayInput(value: string): void {
       this.$emit('input', value);
     },
 
+    /**
+     * Handles the behavior of the Labeled Input when given focus.
+     * @see labeled-form-element.ts mixin for onFocusLabeled()
```

Comment:
I though that making use of `@see` might be a good way to communicate that some methods or props are defined in a mixin.